numba/tests/test_import.py

```diff
@@ -58,7 +58,9 @@ def test_no_accidental_warnings(self):
         # checks that importing Numba isn't accidentally triggering warnings due
         # to e.g. deprecated use of import locations from Python's stdlib
         code = "import numba"
-        flags = ["-Werror",]
+        # See: https://github.com/numba/numba/issues/6831
+        # bug in setuptools/packaging causing a deprecation warning
+        flags = ["-Werror", "-Wignore::DeprecationWarning:packaging:"]
```

Comment:
I tested it, and it only works without the module or when you add `.version`.
```suggestion
        flags = ["-Werror", "-Wignore::DeprecationWarning:packaging.version:"]
```

Reply:
Thanks very much, I've added the `.version` in 94fc06e, hopefully that should now work?

Reply:
https://build.opensuse.org/package/show/home:bnavigator:branches:devel:languages:python:numeric/python-numba

Reply:
Thanks for the build link, I presume https://build.opensuse.org/package/live_build_log/home:bnavigator:branches:devel:languages:python:numeric/python-numba:test/openSUSE_Tumbleweed/x86_64 means success on this particular item?

The `ppc64le` failure is https://github.com/numba/numba/issues/6832 again.
The `i586` failure is strange, `ldexp(2.5, -2)` should give 0.625 but for some reason it's giving `inf`, IIRC it's just a call through to `libm`.

Reply:
Yes, `x86_64` shows that the import test does not fail anymore, thanks to your patch.

I think you mean `i586` is #6832.
I would not be too concerned about ppc64le. Many numeric packages don't build successfully for those platforms.

Reply:
> I think you mean `i586` is #6832.
> I would not be too concerned about ppc64le. Many numeric packages don't build successfully for those platforms.

Yes, apologies, got those transposed. `ppc64le` conda builds are working https://anaconda.org/numba/numba/files so I guess it must be something different about the set up.